core_lib_generator/config_collectors/cache.py

```diff
@@ -1,32 +1,37 @@
 import enum
 
-from core_lib.helpers.shell_utils import input_enum, input_int, input_str
+from core_lib.helpers.shell_utils import input_enum, input_int, input_str, input_yes_no
 
 
-def generate_cache_template() -> dict:
-    cache_name = input_str('Enter name for your cache')
-    cache_type = input_enum(CacheTypes, 'From the following list, what cache will you use?', CacheTypes.Memory.value)
+def generate_cache_template() -> list:
+    cache_list = []
 
-    if cache_type == CacheTypes.Memory.value:
-        print(f'Cache type {CacheTypes(cache_type).name}')
-        return {
-            'config': {
-                cache_name: {
+    add_cache = True
+    while add_cache:

```

Comment:
are we also generating the cache handlers in the CoreLib class?

Reply:
```python
CoreLib.cache_registry.register('memorycache', CacheHandlerRam())
        CoreLib.cache_registry.register('rediscache', CacheHandlerRedis(build_url(**self.config.core_lib.example_core_lib.rediscache.cache.url)))
```
we are generating this

Reply:
Good work!

Reply:
thankyou